src/SetPath.py

```diff
@@ -8,7 +7,7 @@ def SetPathLoadAndSave (Geometry):
         case "Box":
             urdf_path ="./models/humanModelTemplate/humanModelTemplate_shoulder_Box.urdf"
     
-    output_file = "./models/humanModelTemplate/humanModelGenerated.urdf"
+    output_file = "./models/humanModels/" + NameFile + "humanModel.urdf"
     dummy_file = 'no_gazebo_plugins.urdf'
 
-    return urdf_path, output_file ,dummy_file
+    return urdf_path, output_file ,dummy_file
```

Comment:
It is better to add always a newline at the end of the file https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline